app/module/App/src/Controller/StatisticsController.php

```diff
@@ -87,7 +87,7 @@ public function indexAction(array $params)
      */
     private function extractDate(array $params): DateTime
     {
-        $month = $params['month'] ?? null;
```

Comment:
Passing a `null` value to `DateTime` has been deprecated since PHP 8.1. An empty string should be used instead.